src/definition.cpp

```diff
@@ -395,13 +395,13 @@ void DefinitionImpl::addSectionsToIndex()
 
 void DefinitionImpl::writeDocAnchorsToTagFile(TextStream &tagFile) const
 {
-  if (!m_impl->sectionRefs.empty())
-  {
+#define VECTOR_CONTAINS(x, y) (std::find(x.begin(), x.end(), y) != x.end())
+  if (!m_impl->sectionRefs.empty()) {
     //printf("%s: writeDocAnchorsToTagFile(%d)\n",qPrint(name()),m_impl->sectionRef.size());
```

Comment:
Please keep also consistent layout, in this case the `{` bracket not at the end of the line but on the next line.
(multiple places)